### PR TITLE
MUMPS_seq: use OpenBLAS32

### PR DESCRIPTION
**Pull request by dpo**
This library is currently only used by Ipopt, which uses 32-bit Ints and segfaults if MUMPS_seq is linked against `libopenblas64_`.

### CODE REVIEW

**Comment by ViralBShah** [comment]
Perhaps the link line needs `-L${prefix}/lib -lopenblas`. I wonder if we should rename this 32-bit library file name to libopenblas32.

File: M/MUMPS_seq/build_tarballs.jl [hunk 1]
```diff
@@ -43,7 +25,7 @@ make_args+=(OPTF=-O
             CC="$CC -fPIC ${CFLAGS[@]}"
             FC="gfortran -fPIC ${FFLAGS[@]}"
             FL="gfortran -fPIC"
-            LIBBLAS=${OPENBLAS})
+            LIBBLAS="-L${prefix}/lib -lopenblas")
```
Comment (ViralBShah) [inline M/MUMPS_seq/build_tarballs.jl:28]: @giordano Since this is failing on windows, should it be ${libdir}?

Comment (dpo) [reply M/MUMPS_seq/build_tarballs.jl:28]: I wonder how come it was working with libopenblas64_.

Comment (giordano) [reply M/MUMPS_seq/build_tarballs.jl:28]: Yes, using `${libdir}` is definitely a good idea in any case

**Comment by dpo** [comment]
Oops. Didn't see that you made commits here. Something was missing anyways.

**Comment by ViralBShah** [comment]
I noticed that the optimization level is set to `-O`. Should we bump the optimization level to `-O2` or -`O3` once things are working?

**Comment by dpo** [comment]
Sure!

**Comment by giordano** [comment]
See the [warnings](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=2790) produced by this build.  In particular:
```
┌ Warning: To ensure that the correct version of libgfortran is found at runtime, add the following entry to the list of dependencies of this builder
│ 
│     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
└ @ BinaryBuilder /depot/packages/BinaryBuilder/Ay3py/src/auditor/compiler_abi.jl:4
```

**Comment by dpo** [comment]
I always miss those warnings.

**Comment by giordano** [comment]
If you go to the [summary page of the build](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=2790) it's hard to miss them now :wink:

**Comment by dpo** [comment]
Thanks. That's where I should look. I tend to just click on the individual builds directly in the PR.

**Comment by giordano** [comment]
The list of all warnings is a new feature, see #780 :wink:

**Comment by dpo** [comment]
I wanted to suggest such a feature but didn't dare...

**Comment by ViralBShah** [comment]
The warnings in the log are really cool. Coloring by severity will be amazing if possible

**Comment by ViralBShah** [comment]
@dpo ok to merge?

**Comment by giordano** [comment]
> Coloring by severity will be amazing if possible

Yes, we'd like that too, but unfortunately Azure Pipelines only accepts [warning and error as type](https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash#properties).  But also on the Julia side we only have `@warn` and `@error` to show messages that needs attention.

**Pull request merged**